From d42d546bf54df578d7428bb5262f3057fa665a35 Mon Sep 17 00:00:00 2001
From: Jens Georg <jens.georg@desy.de>
Date: Mon, 28 Apr 2025 07:38:02 +0000
Subject: [PATCH] Disable mapfile generator

Upstream-Status: Inappropriate [embedded specific]

This is host tooling that should not be required to be run in either the build
process natively nor on-target at any point in time
---
 CMakeLists.txt | 21 ---------------------
 1 file changed, 21 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index de45813..1b680d5 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -229,27 +229,6 @@ if(INSTALL_SAMPLE_ADAPTER)
   file(COPY ${PROJECT_SOURCE_DIR}/opcuaAdapter_mapping.xml DESTINATION ${CMAKE_BINARY_DIR}/)  
 endif(INSTALL_SAMPLE_ADAPTER)
 
-##########################################################################################################
-##  Install map file generator
-##########################################################################################################
-
-find_package(Python3 REQUIRED)
-
-# install Python modules to correct platform-dependent directory (if installing to system prefix)
-if("${CMAKE_INSTALL_PREFIX}" STREQUAL "/usr" OR "${CMAKE_INSTALL_PREFIX}" STREQUAL "/usr/local")
-  install(DIRECTORY tools/mapfileGenerator/ctk_opcua_generator_tools
-          DESTINATION ${Python3_STDLIB})
-else()
-  install(DIRECTORY tools/mapfileGenerator/ctk_opcua_generator_tools
-          DESTINATION lib/python${Python3_VERSION_MAJOR}.${Python3_VERSION_MINOR}/site-packages)
-endif()
-
-install(FILES tools/mapfileGenerator/chimeratk-opc-ua-mapfile-generator
-        DESTINATION ${CMAKE_INSTALL_PREFIX}/bin
-        PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE
-                     GROUP_READ GROUP_EXECUTE
-                     WORLD_READ WORLD_EXECUTE)
-
 ##########################################################################################################
 ## Create the config files by filling the correct variables into the template (*.cmake.in).
 ## All variables @VARIABLE@ are replaced with the current values, but they have to be set first....
